networkx/algorithms/bipartite/tests/test_project.py

```diff
@@ -66,6 +69,14 @@ def test_path_weighted_projected_graph(self):
         assert edges_equal(list(P.edges()), [(0, 2)])
         P[0][2]["weight"] = 1
 
+    def test_digraph_conversion(self):
+        G = nx.DiGraph()
+        edges = [(0, 1, 1), (1, 2, 1), (2, 3, 1), (3, 4, 2)]
+        G.add_weighted_edges_from(edges)
```

Comment:
Any reason to use weighted edges here? AFAICT edge weights in the original bipartite graph are not considered in the algorithm. The weights in the output graph are the result of the algorithm.

That actually might be a bit confusing as the node attributes are copied over from the input to the projected graph, but the edge attributes are not (and if there are weighted edges, the meaning of the "weight" attribute changes between the input and output graphs).

Reply:
You're completely correct, made the change. Thanks.